tests/test_samples.py

```diff
@@ -713,12 +713,14 @@ def test_masking():
 
     for kind in kinds:
         fig, axes = make_1d_axes(['x0', 'x1', 'x2'])
-        pc[mask].plot_1d(axes=axes, kind=kind + '_1d')
+        pc.loc[mask].plot_1d(axes=axes, kind=kind + '_1d')
```

Comment:
This is another case, where I'd prefer changing the mask to an array. When I see `.loc` I expect one of the labels for indexing...
```python
mask = pc['x0'].to_numpy() > 0
```